cylc/flow/data_messages_pb2.py

```diff
@@ -1,37 +1,47 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
+# NO CHECKED-IN PROTOBUF GENCODE
 # source: data_messages.proto
-# Protobuf Python Version: 4.25.3
+# Protobuf Python Version: 5.27.0
```

Comment:
This is outside of the version range specified in `setup.cfg`, which is why I think you're getting import errors

https://github.com/cylc/cylc-flow/blob/450728ad966dcc83dcfecb2d857c51567aa22d11/setup.cfg#L74

Reply:
(The dependency range in `setup.cfg` can be widened if we test that it maintains compatibility (test that old UIServers can operate with new workflows and vice versa)